cabal-testsuite/PackageTests/WorkingDir/working-dir-test.cabal

```diff
@@ -0,0 +1,18 @@
+cabal-version:       2.4
+name:                working-dir-test
+version:             0.1.0.0
+synopsis:            Test that we don't pass --working-dir when unsupported
+license:             BSD-3-Clause
+author:              NA
+maintainer:          NA
+category:            Testing
+build-type:          Custom
+
+custom-setup
+  setup-depends: base, Cabal >= 2.4 && <= 3.12

```

Comment:
Is `<= 3.12` intended?  Usually, `<=` is not good for upper bounds.  Should this be `< 3.13`?